src/CameraControls.ts

```diff
@@ -1117,6 +1122,58 @@ export class CameraControls extends EventDispatcher {
 
 		};
 
+		const lockPointer = (): void => {
+
+			if ( ! this._enabled || ! this._domElement ) return;
+
+			// Element.requestPointerLock is allowed to happen without any pointer active - create a faux one for compatibility with controls
+			const pointer: PointerInput = {
+				pointerId: 1,
+				clientX: 0,
+				clientY: 0,
+				deltaX: 0,
+				deltaY: 0,
+				mouseButton: null
+			};
+			this._activePointers.push( pointer );
```

Comment:
Shouldn't there be only one pointer during pointer lock?
If so, we can call `this.cancel()` to remove all this._activePointers before this line.

Reply:
Yes! Also - one bug I noticed, when pointer is locked and you click (ie. trigger one of the onPointer events), all the `dragging` chain is executed - and the last step of it is to remove `pointermove` listeners.

Is there a clean way we can avoid that - or would just checking for pointer lock make sense?

Reply:
Ah right.
I think there is no clean way for this particular case.
So, I would do:

> would just checking for pointer lock make sense?